index.js

```diff
@@ -15,11 +15,15 @@ async function subscribe(topic_name, subscription_name, subscriber) {
     }
   }
 
-  const subscription = pubsub.subscription(subscription_name, {
+  // Default subscriber options
+  // Documentation here https://cloud.google.com/pubsub/docs/pull#config
+  const defaultOptions = {
     flowControl: {
       maxMessages: 10,
     },
-  });
+  };
+
+  const subscription = pubsub.subscription(subscription_name, options || defaultOptions);
```

Comment:
An empty object evaluates to true in JS. So, if options is an empty object `{}`, we will end up passing it to the subscription function instead of the `defaultOptions`, is that what we want?

Reply:
We can discuss what to do with this type of input. If the options is an empty object `{}`, the Pubsub library will use it's own default options, instead of the `defaultOptions` in this wrapper.  This is not expected behavior. Maybe we can have a check to see `(if subscriberOptions.ackDeadline || subscriberOptions.batching || subscriberOptions.flowControl || subscriberOptions.streamingOptions)`, then we use the subscriberOptions. Otherwise, use defaultOptions. What do you think?

Reply:
If the Pubsub library will use it's default options when the passed options is an empty `{}`, then I think that's okay.

Reply:
Ok perfect 👍